driver/stm32/lvstm32.py

```diff
@@ -19,7 +19,11 @@ def task_handler(self, _):
         lv.task_handler()
 
     def timer_cb(self, t):
-        lv.tick_inc(self.delay)
-        # Passing self.task_handler would cause allocation.
-        micropython.schedule(self.task_handler_ref, 0)
+	try:
```

Comment:
What exceptions are you catching here?  
I don't think neither `lv.tick_inc` nor `micropython.schedule` are expected to throw exceptions.  

btw there are also some problems with your indentation.

Reply:
Hi @amirgon 
I see this
 `>>> import lvgl_stm32
uncaught exception in Timer(4) interrupt handler
RuntimeError: `

sorry for indentation. I'm using gedit or CubeMXIDE that use tab instad of spaces... How can I easily fix this?

Reply:
I'm not sure about the editors, I'm using Vim. Just copy the spaces from other lines.  

About the exception - usually it comes with a traceback with more information.  
If you don't see any traceback, please try adding the line  
```python
micropython.alloc_emergency_exception_buf(100)
```
right after the imports, and see if you get the traceback.

Reply:
When calling `micropython.schedule(self.task_handler_ref, 0)` the exception was `schedule queue full`. I found the issue was due to long processing in modlcd now fixed.
I restored slvstm32.py